contextual-inquiries/contextual_inquiry_1.typ

```diff
@@ -0,0 +1,198 @@
+==== Contextual Inquiry 1 <ci_1>
+
+- Teilnehmer: Patrick
+- Datum: 09.10.2024
+- Ort: In seinem Zimmer
+- Dauer: [Unsure]
+
+==== Generelle Fragen
+
+/ Seit wann studierst du?:
+Die Testperson hat ihr Studium im Herbstsemester 2021 gestartet.
+
+/ In welchem Studienmodell (TZ2/3 oder Vollzeit, unterschiedlich)?:
+Sie hat für 1 Semester vollzeit studiert. Alle anderen Semester im TZ2-Modell.
+
+/ In welchem Semester bist du gerade?:
+Sie ist aktuell im 7. Semester.
+
+/ Wann planst du deinen Abschluss?:
+Sie hat seinen Abschluss für das Frühjahrsemester 2025 geplant.
+Sie schliesst also nach X Semestern ab.
+
+/ Woher kennst du den Semesterplaner schon?:
+Der Semesterplaner wurde von Studenten und Dozenten empfohlen.
+
+/ Wann nutzt du den Semesterplaner typischerweise?:
+Sie nutzt den Semesterplaner bei der Modulanmeldung, um den Stand seiner Studien abzufragen, aber auch zum Spass.
+Sie passt den Semesterplaner meist an, wenn die Modulauswahl in der 1. Phase bereits relativ festgelegt ist.
+
+/ Wie häufig nutzt du den Semesterplaner?:
+Sie nutzt den Semesterplaner etwa monatlich.
+Während der Modulanmeldung nutzt er ihn häufiger, etwa 5 Mal, über mehrere Tage verteilt.
+
+/ Hast du in der Vergangenheit auf eine andere Weise dein Studium geplant als mit dem Semesterplaner? Wenn ja, wie?:
+Sie hat seine ersten zwei Semester nach dem Musterstudienplan geplant.
+
+/ Planst du eine Vertiefung?:
+Zu Beginn des Studiums hatte sie die Vertiefung "Software Engineering" verfolgt, mittlerweile aber nicht mehr.
+Der Grund sei, dass ihr doch nicht alle Module der Vertiefung interessieren.
+
+/ Folgst du einem Musterstudienplan? Wenn ja/nein, weshalb?:
+Sie folgt grösstenteils dem Musterstudienplan für Teilzeit "Software Engineering", weicht aber leicht ab.
+Der Grund sei, dass ihr doch nicht alle Module des Musterstudienplanes interessieren.
+
+/ Arbeitest du momentan neben dem Studium oder hast du mal neben dem Studium gearbeitet? Wenn ja, in welchem Pensum?:
+Ja, sie arbeitet zu 60%.
+
+
+==== Überprüfen des bisherigen Plans
+
+/ Wo/wie hast du den Plan gespeichert?:
+Sie hat die URL zu seinem Plan in einem Bookmark gespeichert.
+
+/ Prüfst du den Plan? Wenn ja, wie und auf was?:
+[Schaut URL an]
```

Comment:
yes, er het gseit er luegt t url a wegem startsemester woner immer setze muess will ers so abgspeicheret het ohni. und dass denn die it url inechunnt bim uswähle devo. meh prüeft er nöd, het nüt dezue gseit

Reply:
Ich han 0 Plan, wie das ufschriebe

Reply:
```suggestion
Setzt das Startsemester, welches im abgespeicherten Link nicht gesetzt war. Prüft danach, ob das Startsemester in der URL korrekt erscheint und ob die erreichten ECTS korrekt aktualisiert werden.
```

Reply:
theoretisch chönnt mer au s ganze als bulletpoints ufliste und halbsätz mache, denn fallts weniger uf und isch eifacher zum lese, au für eus zum uswerte